base/src/com/google/idea/blaze/base/sync/SyncPhaseCoordinator.java

```diff
@@ -362,6 +362,24 @@ void runSync(BlazeSyncParams params, boolean singleThreaded, BlazeContext contex
           projectState != null
               ? BuildPhaseSyncTask.runBuildPhase(project, params, projectState, buildId, context)
               : BlazeSyncBuildResult.builder().build();
+      //If we have partial success we'll run the build again this time only with intellij-info-java
+      if ( projectState != null && syncResultFromBuildPhase(buildResult, context) == SyncResult.PARTIAL_SUCCESS) {
```

Comment:
nit: format

Reply:
i'm ok with nits. Which format? I don't want to format the whole file since I think my formatter isn't defined like theirs. You mean the space after the `(` but no space before `)`? Something else?

Reply:
ended up removing the before space since that's what I saw they do in other places

Reply:
yes, I meant spaces. For Google Java stuff it makes sense to use their code style, if we plan to contribute back.

Reply:
https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml

Reply:
I agree. Concretely here I think it’s ok now.